src/AppBundle/Tests/Subscriber/StatusChangeByReviewTest.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+namespace AppBundle\Tests\Subscriber;
+
+use AppBundle\Event\GitHubEvent;
+use AppBundle\GitHubEvents;
+use AppBundle\Issues\Status;
+use AppBundle\Repository\Repository;
+use AppBundle\Subscriber\StatusChangeByReviewSubscriber;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+
+class StatusChangeByReviewSubscriberTest extends \PHPUnit_Framework_TestCase
+{
+    private $statusChangeSubscriber;
+
+    private $statusApi;
+
+    private $repository;
+
+    /**
+     * @var EventDispatcher
+     */
+    private static $dispatcher;
+
+    public static function setUpBeforeClass()
+    {
+        self::$dispatcher = new EventDispatcher();
```

Comment:
this should not be a static property. The dispatcher instance must not be shared between tests, otherwise the listener will run several times for subsequent tests (as listeners registered by previous tests will still be here).
I faced this issue when implementing the same feature before seeing your PR, because I added a check in the subscriber to avoid changing the status if it is already the current one. (making the test fail when a previous listener was running too with the config of the previous test).